core/ir/aarch64/instr_create_api.h

```diff
@@ -1693,6 +1693,56 @@ enum {
 #define INSTR_CREATE_bif_vector(dc, Rd, Rm, Rn) \
     instr_create_1dst_2src(dc, OP_bif, Rd, Rm, Rn)
 
+/**
+ * Creates a FCVTAS vector instruction.
+ * \param dc      The void * dcontext used to allocate memory for the instr_t.
+ * \param Rd      The output register.
+ * \param Rm      The first input register.
+ * \param width   The input element vector width.
```

Comment:
Hmm the type of these operands might be confusing to newbies, and to me: snooping around I think this takes an immed int opnd whose type is one of the VECTOR_ELEM_WIDTH_* constants, right?  Something like OPND_CREATE_BYTE()?  Maybe add a doxygen pointer to these constants or macros?

Reply:
Done